dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-pytorch.ts

```diff
@@ -93,7 +93,7 @@ export function usePytorch(model: { [field: string]: any }): IJsonItem[] {
     },
     {
       type: 'input',
-      field: 'pythonCommand',
+      field: 'pythonLauncher',
       name: t('project.node.pytorch_python_command'),
```

Comment:
Please change `project.node.pytorch_python_command` in both chinese and english.

Reply:
Done. Thank you for review.